### DISCUSSION TITLE
Chat Card demo not saving data

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#927] [Q&A] [answered]
Here is a quick recording showing what I am seeing. Is there something I am missing here? It seems the data should be stored in the card in the form of a dictionary but doesn't seem to be. 

https://user-images.githubusercontent.com/78372056/126512631-7e572439-f675-4a97-ad46-84466fd0054d.mp4

**Comment by lo5** [comment, marked as answer]
This feature is not GA yet. It works only on dashboards, and only if `waved` is started with the `-editable` flag.

**Reply by Alex-Wenner-FHR** [reply]
Thanks for the clarification! This is a cool feature. Would be really cool if you could extract the data (assuming you will be able to anyways).
